tutorials/RobustTestingExample.ipynb

```diff
@@ -0,0 +1,4480 @@
+              "      filter: drop-shadow(0px 1px 2px rgba(0, 0, 0, 0.3));\n",
+              "      fill: #FFFFFF;\n",
+              "    }\n",
+              "  </style>\n",
+              "\n",
+              "      <script>\n",
+              "        const buttonEl =\n",
+              "          document.querySelector('#df-d7c834cc-25fb-41ec-9215-1b3e142d0ab3 button.colab-df-convert');\n",
+              "        buttonEl.style.display =\n",
+              "          google.colab.kernel.accessAllowed ? 'block' : 'none';\n",
+              "\n",
+              "        async function convertToInteractive(key) {\n",
+              "          const element = document.querySelector('#df-d7c834cc-25fb-41ec-9215-1b3e142d0ab3');\n",
+              "          const dataTable =\n",
+              "            await google.colab.kernel.invokeFunction('convertToInteractive',\n",
+              "                                                     [key], {});\n",
+              "          if (!dataTable) return;\n",
+              "\n",
+              "          const docLinkHtml = 'Like what you see? Visit the ' +\n",
+              "            '<a target=\"_blank\" href=https://colab.research.google.com/notebooks/data_table.ipynb>data table notebook</a>'\n",
+              "            + ' to learn more about interactive tables.';\n",
+              "          element.innerHTML = '';\n",
+              "          dataTable['output_type'] = 'display_data';\n",
+              "          await google.colab.output.renderOutput(dataTable, element);\n",
+              "          const docLink = document.createElement('div');\n",
+              "          docLink.innerHTML = docLinkHtml;\n",
+              "          element.appendChild(docLink);\n",
+              "        }\n",
+              "      </script>\n",
+              "    </div>\n",
+              "  </div>\n",
+              "  "
+            ]
+          },
+          "metadata": {},
+          "execution_count": 19
+        }
+      ],
+      "source": [
+        "test_df.head(2)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "a0b7f69e",
+      "metadata": {
+        "id": "a0b7f69e"
+      },
+      "source": [
+        "As these two features are independent, no linear combinations will be found, so the test will pass"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 20,
+      "id": "7d6cd86c",
+      "metadata": {
+        "id": "7d6cd86c"
+      },
+      "outputs": [],
+      "source": [
+        "LinearCombinationsTest(test_df).run()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "93560e00",
+      "metadata": {
+        "id": "93560e00"
+      },
+      "source": [
+        "Let's artificially generate a dataset with a column which is linearly dependent with `age`. This aims to serve as an example when we have a dataset with clear feature dependencies, which should be removed prior to model trainings."
```

Comment:
I think that here you want you add a new feature to the dataset and run the test again to prove that it will raise an exception, but that part is missing

Reply:
The chuck of code is below the next section (label leaking test)